cfme/scripting/bz.py

```diff
@@ -44,18 +44,19 @@
 BZ_URL = conf.env.bugzilla.url
 
 
-def get_report(directory):
+def get_report(directory, inc_manual=True):
```

Comment:
@prichard77 since this is dependent on the bugzilla credentials being correct, and this is the source method for all the data that the rest of the module is using, I think we should check here whether or not credentials are available.

The instance given by `cfme.utils.bz.Bugzilla.from_config` will have either the `username` or the `api_key` attribute set if credentials were provided.

If the credentials are wrong, that a whole other issue that I don't think you need to handle here.

Reply:
Updated as suggested.

Reply:
Good catch too on the instance attribute being `key`, even though the kwarg is `api_key`